product_pricelist_supplierinfo/models/product_template.py

```diff
@@ -13,40 +13,12 @@ def _get_supplierinfo_pricelist_price(
             self, rule, date=None, quantity=None, product_id=None):
         """Method for getting the price from supplier info."""
         self.ensure_one()
+        product = self.product_variant_id
```

Comment:
But... If i have more than one...???

Reply:
hmm... then you should not use template to get the variant price? I mean, if you have variant based prices, what price are you expecting to get when asking for template price?